analysis/Scripts/MC_job_Entries.py

```diff
@@ -18,19 +18,19 @@
 for job in parsJob:
```

Comment:
what is the purpose of this script?

Reply:
If I remember correctly it was for the presentation of Jacco to show that the MC after the cuts was insufficiant for our purposes